modules/tensor_mechanics/doc/content/source/materials/ComputePlaneSmallStrain.md

```diff
@@ -4,27 +4,29 @@
 
 ## Description
 
-The material `ComputePlaneSmallStrain` calculates the small incremental
-strain for 2D plane strain problems. It can be used for  classical
-[thick body plane strain](https://en.wikipedia.org/wiki/Plane_stress)
-problems, [Weak Plane Stress](/WeakPlaneStress.md) models, or in
+The material `ComputePlaneSmallStrain` calculates the small total
+strain for 2D plane strain problems. It can be used for classical
+[plane strain or plane stress](https://en.wikipedia.org/wiki/Plane_stress)
+problems, or in
 [Generalized Plane Strain](tensor_mechanics/generalized_plane_strain.md) simulations.
 
 ## Out of Plane Strain
 
-In the classical thick body plane strain problem, the strain
-component in the out-of-plane direction (the infinitely thick
-direction) is held constant at zero:
+In the classical plane strain problem, it is assumed that the front and back
+surfaces of the body are constrained in the out-of-plane direction, and that
+the displacements in that direction on those surfaces are zero. As a 
+result, the strain and deformation gradient components in the out-of-plane
+direction are held constant at zero:
```

Comment:
This is the third time I'm reading this same paragraph. Should this just be an "include"?

Reply:
Maybe.  It would probably be best to just have a page with common material that they all reference rather than including the same text so that it is repeated multiple times in the documentation. I would suggest that this is a bigger issue than this minor change -- there's far more repeated text than just what I added here.